pkg/workflow/parse.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workflow
+
+import (
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+)
+
+// NOTE(tony): offical reference
+// https://github.com/argoproj/argo/blob/master/pkg/apis/workflow/v1alpha1/workflow_types.go
+type workflow struct {
```

Comment:
Can we directly use struct `Workflow` in https://github.com/argoproj/argo/blob/master/pkg/apis/workflow/v1alpha1/workflow_types.go ?

Reply:
Absolutely! Thanks for pointing it out.